### ISSUE TITLE
Tooltip mistake in tag edit form

### ISSUE THREAD

**Issue by elvirolo** [mailpile/Mailpile#2238]
Version: 1.0.0rc6

In the tag editing form, the tooltips for "Automatic tagging" and "Tag by hand" should be split.

![screenshot_2019-10-08_12-23-20_098797452](https://user-images.githubusercontent.com/7861012/66389348-b5ce6480-e9b7-11e9-9b06-7a2b3556b021.png)

**Comment by BjarniRunar** [comment]
Good catch, thanks for reporting this! You're almost correct: the tooltip applies only to the "Automatic tagging" option, "Tag by Hand" should have a different tooltip.

I'll get this fixed.
